packages/webpack-plugin/index.multi-schema.js

```diff
@@ -0,0 +1,164 @@
+// @ts-check
```

Comment:
Just a copy for myself for terrible code that could handle multiple clients/schemas.